app/src/scenes/Items/interfaces.ts

```diff
@@ -0,0 +1,35 @@
+export interface ItemStateProps {
+  items: Item[];
+  isLoading: boolean;
+  pagination: {
+    current: number,
+    pageSize: number,
+    total: number,
+  };
+}
+
+export interface Item {
+  id: number;
+  link: string;
+  translations: Language[];
+}
+
+export interface SubCategory {
+  id: number;
+  translations: Language[];
+}
+
+export interface Language {
```

Comment:
Better if we could use something like `SubCategoryTraslation`

Reply:
ok